doc/api/pymongo/pool.rst

```diff
@@ -2,5 +2,6 @@
 ==============================================================
 
 .. automodule:: pymongo.pool
-   :synopsis: Pool module for use with a MongoDB client.
-   :members:
```

Comment:
I've intentionally removed SocketInfo from the docs since it's an internal pymongo API. The updated pool page only contains docs for PoolOptions.